aat/src/aat/resources/features/F-102 - TTL Base Complex Type/S-102.2.td.json

```diff
@@ -0,0 +1,674 @@
+{
+  "_guid_": "S-102.2",
+  "_extends_": "F-102_Test_Complex_Organisation",
+  "title": "must return all details successfully for a case type containing ChangeOrganisationRequest fields",
```

Comment:
Title in scenario TD files should match feature file's corresponding scenario description.